core/Piranha.Manager/assets/src/scss/inc/_variables.scss

```diff
@@ -17,7 +17,6 @@ $indigo:        #6610f2;
 $purple:        #6f42c1;
 $pink:          #e83e8c;
 $red:           #a94441;
-$red-bright:    #f94c3b;
```

Comment:
This should not be renamed. This color is used for validation purposes in other parts of the manager and just not the login.

Reply:
I though I checked for any other occurences of this variable in the code. My mistake.